CDCgov/cdc-open-viz
# Enhancement: Dev-8931 Hide Markup Include on null values

## Enhancement: Dev-8931 Hide Markup Include on null values
[https://websupport.cdc.gov/projects/DEV/issues/DEV-8931](https://websupport.cdc.gov/projects/DEV/issues/DEV-8931)

Added ability to display/hide markup include depending on if there are values with null values.

## Testing Steps

Testing Instructions

Scenario: Upload [dev-8931-hide-markup-include.json](https://github.com/user-attachments/files/17544291/dev-8931-hide-markup-include.json), Click on Dashboard Preview 
Expected: There is a Dashboard Filter and a Markup Include with text

1. Choose Normal Weight from the dropdown.
Expected: Markup Include is shown with no value next to Percentage(s).

2. Choose Not Checked in past 5 Years
Expected: Markup Include is shown with no value next to Class(es).

3. Click Dashboard Description
4. Click the tools icon on the Markup Include
5. Open Content Editor 
6. Click the checkbox next to Hide section on null
7. Open Dashboard Preview

Expected: When Normal Weight or Not Checked in past 5 years is chosen, there shouldn't be any Markup Include showing. If you choose any other option that has values for both variables then the Markup Include will show. 

## Self Review

- I have added testing steps for reviewers
- My changes generate no new warnings

## Additional Notes

In the EditorPanel.tsx file, the only changes I made are from 199-221. The rest of the changes are linter changes.
